GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -42,13 +43,44 @@ class ItemViewControllerSpec: QuickSpec {
             
             describe("viewWillAppear") {
                 it("sets the data source to a static item list") {
+                    
+                    let itemRepository = FakeItemRepository()
+                    itemRepository.stub(items: Item.testData)
+                    controller.itemRepository = itemRepository
```

Comment:
It looks like you are setting `controller.itemRepository` twice. Once in the setup, and once here.

What is the reason for setting it twice?

Reply:
removed line from before block... keeping tests wet for now.